### DISCUSSION TITLE
如何添加站点统计

### DISCUSSION THREAD

**Discussion by Angel0726** [tangly1024/NotionNext#2055] [General]

**Comment by ipatpat** [comment]
https://tangly1024.com/article/notion-next-guide#d7ac96d3889c45998b991bed8282c7e6

**Comment by Angel0726** [comment]
> https://tangly1024.com/article/notion-next-guide#d7ac96d3889c45998b991bed8282c7e6

- 我遇到的问题如下。coolnext.vercel.app与blog.wsxlkj.cf都是vercel上同一个部署的项目。但是实际使用百度统计的跨域名部署之后，两个域名的站点统计不一样。（两个域名的原因是coolnext.vercel.app国内不能访问，所以增加了blog.wsxlkj.cf作为域名）
- 猜测问题原因：如果添加百度站点统计是不是需要将ANALYTICS_BUSUANZI_ENABLE设置为false。
